connexta/eve
# reverted back to run.sh & fixed Docker API version error

#### What does this PR do?
Fixed Docker API version error and reverted back to run.sh for Dockerfile in order to fix Jenkins

#### Who is reviewing it?

@vinamartin @Kjames5269 @bankent1 @alchucam @mcalcote

#### How should this be tested?
Check Jenkins console output 
For Deploying to Marathon Service, I'm currently waiting on IT team to setup DCOS service.

#### Screenshots

![image](https://user-images.githubusercontent.com/40583917/60616829-38c39b80-9d87-11e9-8ec9-fbd076aa7b75.png)
I have tested on my own github repo. 



#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
